data/tokens.json

```diff
@@ -1162,6 +958,18 @@
         "value": "{gtmds_color_red-1}",
         "type": "color"
       }
+    },
+    "gtmds_opacity-hover": {
+      "value": "0.08",
+      "type": "other"
+    },
+    "gtmds_opacity-selected": {
+      "value": "0.24",
+      "type": "other"
+    },
+    "gtmds_opacity-selected-hover": {
+      "value": "0.32",
+      "type": "other"
```

Comment:
What's the use case for that one?

Reply:
these are the opacity values detached from the hover/selected whatever but intended for them just like the color that we apply opacity to